<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PVRDecrypt GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PVRDecrypt.qrc">
    <normaloff>:/decrypt_icon</normaloff>:/decrypt_icon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Tasklist</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidget">
         <property name="rowCount">
          <number>0</number>
         </property>
         <column>
          <property name="text">
           <string>Image</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>File name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Channel</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rect time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Duration</string>
          </property>
          <property name="textAlignment">
           <set>AlignRight|AlignVCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Imagedefinition</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>File size</string>
          </property>
          <property name="textAlignment">
           <set>AlignRight|AlignVCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>X</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QPushButton" name="browseOutputFolderpushButton">
         <property name="text">
          <string>Browse</string>
         </property>
         <property name="icon">
          <iconset resource="PVRDecrypt.qrc">
           <normaloff>:/folder-upload-icon</normaloff>:/folder-upload-icon</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="selectOutputFolderLineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QFrame" name="frame_3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="stopPushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
         <property name="icon">
          <iconset resource="PVRDecrypt.qrc">
           <normaloff>:/stop</normaloff>:/stop</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="decryptPushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Decrypt</string>
         </property>
         <property name="icon">
          <iconset resource="PVRDecrypt.qrc">
           <normaloff>:/decrypt</normaloff>
           <disabledoff>:/decrypt_disabled</disabledoff>:/decrypt</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionDecrypt"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionRemove"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_files"/>
    <addaction name="actionBrowseOutputFolder"/>
    <addaction name="actionOpen_out_folder"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar1"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_files"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
   <addaction name="actionDecrypt"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionBrowseOutputFolder"/>
   <addaction name="actionMetaWindow"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="PVRDecrypt.qrc">
     <normaloff>:/exit-icon</normaloff>:/exit-icon</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="PVRDecrypt.qrc">
     <normaloff>:/icons/system-help.png</normaloff>:/icons/system-help.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDecrypt">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="PVRDecrypt.qrc">
     <normaloff>:/decrypt_icon</normaloff>
     <disabledoff>:/decrypt_disabled</disabledoff>
     <disabledon>:/decrypt_disabled</disabledon>:/decrypt_icon</iconset>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionBrowseOutputFolder">
   <property name="icon">
    <iconset resource="PVRDecrypt.qrc">
     <normaloff>:/folder-upload-icon</normaloff>:/folder-upload-icon</iconset>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="PVRDecrypt.qrc">
     <normaloff>:/edit-remove-icon</normaloff>:/edit-remove-icon</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAdd_files">
   <property name="icon">
    <iconset resource="PVRDecrypt.qrc">
     <normaloff>:/download-folder-icon</normaloff>:/download-folder-icon</iconset>
   </property>
   <property name="text">
    <string>Add files</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="PVRDecrypt.qrc">
     <normaloff>:/stop</normaloff>:/stop</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpen_out_folder">
   <property name="text">
    <string>Open out folder</string>
   </property>
  </action>
  <action name="actionMetaWindow">
   <property name="icon">
    <iconset resource="PVRDecrypt.qrc">
     <normaloff>:/help-about-icon</normaloff>:/help-about-icon</iconset>
   </property>
   <property name="text">
    <string>Meta</string>
   </property>
   <property name="toolTip">
    <string>Meta Informationen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PVRDecrypt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>decryptPushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionDecrypt</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1153</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopPushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionStop</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>977</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseOutputFolderpushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionBrowseOutputFolder</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1064</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
